### ISSUE TITLE
Expose state for screen readers on multiple occasions.

### ISSUE THREAD

**Issue by HenrikHartiala** [HSLdevcom/jore4#1441]
Part of accessibility audit findings

For example choosing priorities in search etc. We need to add 'aria-selected' to expose the state to screen readers. We currently might have something like 'data-selected' but that seems to be a custom value not fit for this use.

- State is not exposed
   - [ ] Missing state for screen readers ([24](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g24d8c9308d3_0_89))
   - [ ] (Routes) Displayed item count buttons' don't show state to screen readers ([33](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g252dad890e8_0_0))
   - [ ] Accordion doesn't expose state to screen reader, see slide 21 ([78](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g2520fb25fd9_0_104))
   - [ ] (Lines&Routes Search) Filters don't expose state, see slide 21 ([86](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g252dad890e8_0_10))
   - [ ] (Forms?) Drawer toggle is missing title and state information for screen readers ([62](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g2517a945953_3_55))
   - [ ] (Forms) No field state for screen readers (see slides 24, 33 and their solutions) ([64](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g251896a5555_0_10))
